### PR TITLE
Labs 0, 1, 2 review

### PR DESCRIPTION
**Pull request by kylelee24**
Lab 0:

- Added TOC

Lab 1:

- Screenshot images
- Wording/grammar

Lab 2:

- Screenshot images
- Enable API pre-requisite (CLI commands)
- Missing $PROJECT_ID declaration
- Wording/grammar
- Used project name for bucket within `backend.tf` file

Note: Considering at the end of each lab, adding a link to the next lab.

### CODE REVIEW

File: docs/lab_2.md [hunk 1]
```diff
@@ -41,48 +55,60 @@ gcloud projects add-iam-policy-binding $PROJECT_ID \
...
 gsutil versioning set on gs://${PROJECT_ID}-tfstate
 ```
 
-4. Replace the PROJECT_ID placeholder with the project ID in the backend.tf
+4. In GitHub, edit the `backend.tf` file and update the `bucket` to the Cloud Storage bucket that was just created (E.g., `sc-gc-devops-bootcamp-tfstate`).
+
+```json
+terraform {
+  backend "gcs" {
+    bucket = "sc-gc-devops-bootcamp-tfstate"
```
Comment (philbenjamin) [inline docs/lab_2.md]: Instruct the user to update the bucket to ${PROJECT_ID}-tfstate (otherwise bucket names wouldn't be unique)

**Review by philbenjamin — CHANGES_REQUESTED** [review]
Change the bucket name on backend.tf to PROJECT_ID-tfstate and instruct user to substitute their actual project id and commit

File: docs/lab_2.md (hunk 1 above)
Comment (kylelee24) [reply docs/lab_2.md]: So the storage bucket names need to be unique, shouldn't we suggest to use something other than the `${PROJECT_ID}` or at least append a unique hash? I'd expect at least a few users will use the same project name that we used in the documentation/gif.

Comment (kylelee24) [reply docs/lab_2.md]: @philbenjamin, I have updated the documentation to generate and add a random string (lowercase letters + numbers) to the bucket name. This should resolve any unique bucket creation name issues that may occur.

```bash
PROJECT_ID=$(gcloud config get-value project)
RAND=$(head /dev/urandom | tr -dc a-z0-9 | head -c 13)
gsutil mb gs://${PROJECT_ID}-${RAND}-tfstate
```

Comment (philbenjamin) [reply docs/lab_2.md]: Hey @kylelee24 , just saw this. So the project-id's in Google are globally unique. They are different from Project names which are not necessarily unique. So the random string addition is not necessary

Comment (kylelee24) [reply docs/lab_2.md]: Hey @philbenjamin, my mistake. I thought the project id = project name. Looking at it, I see that the project id will be the same as the project name if the ID does not already exist. However if it exists, it appends a series of numbers after (E.g., `sc-gc-devops-bootcamp-284719`). I made the changes to ask the user to update the placeholder value of `PROJECT_ID` with the value of `${PROJECT_ID}`. I also updated the `backend.tf` file to have the `PROJECT_ID` placeholder.

**Review by philbenjamin — APPROVED** [review]
mergin in kyle's updates to lab0,1,2

**Pull request merged**